packages/itwin/measure-tools/src/measurements/AreaMeasurement.ts

```diff
@@ -433,18 +463,26 @@ export class AreaMeasurement extends Measurement {
           areaSpec !== undefined
             ? { value: this._polygon.area, formatSpec: areaSpec }
             : undefined,
-      },
-      {
-        label: MeasureTools.localization.getLocalizedString(
-          "MeasureTools:tools.MeasureArea.popupAreaXY"
-        ),
-        name: "AreaMeasurement_AreaXY",
-        value: fAreaXY,
-        aggregatableValue:
-          areaSpec !== undefined
-            ? { value: this._polygon.areaXY, formatSpec: areaSpec }
-            : undefined,
-      },
+      }
+    );
+
+    if (this._sheetToWorldScale === undefined) {
```

Comment:
Can you elaborate why we need to check for sheet to world scale? Seems we can compute the area either case.

Reply:
We do not want to display the XY area if it's a sheet measurement since there's no point, it's in 3d so any area is XY anyway.
If we really wanted to show it, it would always display as 0 for cross sections and would always be equal to the regular area for plan views (since there is no Z depth).